### PR TITLE
Candid keywords

### PR DESCRIPTION
**Pull request by bdemann**

### CODE REVIEW

File: examples/candid_types/.gitignore
Comment (lastmjs) [inline examples/candid_types/.gitignore:1]: This example should be named `candid_keywords`

File: examples/candid_types/test/tests.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import { Test } from 'azle/test';
+import { _SERVICE } from './dfx_generated/candid_types/candid_types.did';
+import { ActorSubclass } from '@dfinity/agent';
+
+export function getTests(candid_canister: ActorSubclass<_SERVICE>): Test[] {
+    return [
+        {
+            name: 'test query with keyword name',
+            test: async () => {
+                await candid_canister.opt();
```
Comment (lastmjs) [inline examples/candid_types/test/tests.ts:10]: Why not get the result and test that it's correct?

File: examples/candid_types/package.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+    "scripts": {
+        "pretest": "ts-node --transpile-only --ignore=false test/pretest.ts",
+        "test": "ts-node --transpile-only --ignore=false test/test.ts"
+    },
+    "dependencies": {
+        "azle": "0.17.1"
+    },
+    "devDependencies": {
+        "@dfinity/agent": "0.11.1",
```
Comment (lastmjs) [inline examples/candid_types/package.json:10]: These dependencies should be up-to-date

**Review by lastmjs — CHANGES_REQUESTED** [review]

File: examples/candid_types/test/tests.ts (hunk 1 above)
Comment (bdemann) [reply examples/candid_types/test/tests.ts:10]: Because if it fails it will fail before we get the result, the result is not necessary

**Review by lastmjs — APPROVED** [review]

**Pull request merged**